app/src/main/java/com/guerinet/mymartlet/util/dbflow/databases/CoursesDB.java

```diff
@@ -46,15 +47,12 @@
      * @param callback Optional callback called when the transaction is finished
      */
     public static void setCourses(Term term, List<Course> courses,
-            @Nullable DBUtils.Callback callback) {
-        if (courses == null) {
-            courses = new ArrayList<>();
-        }
+                                  @Nullable DBUtils.Callback callback) {
+
+        if (courses == null) courses = new ArrayList<>();
 
         // Set the term on the passed list of courses
-        for (Course course : courses) {
-            course.setTerm(term);
-        }
+        courses.forEach(course -> course.setTerm(term));
```

Comment:
I know this is a Java 8 feature... Did they backport this to pre-Marshmallow? 
Also quick note while I'm talking Android versions: I'm bringing back the min version to 19. After we switched to Lollipop a couple of weeks ago I had a some users send me emails asking to bring it back down.